tests/lib/rules/prop-types.js

```diff
@@ -1486,6 +1486,7 @@ ruleTester.run('prop-types', rule, {
         '  }',
         '}'
       ].join('\n'),
+      settings: {react: {flowVersion: '0.52.0'}},
```

Comment:
I suppose it depends on the next `eslint-plugin-react` version? Major version bump means we can make the default to 0.53. Minor version bump means we need to stick to supporting 0.52 as the default?

BTW, the current default for React (and also in this PR for Flow) is `999.999.999`. So it just assumes by default "the latest version" if the setting is not specifically set.